cpp/src/io/parquet/page_enc.cu

```diff
@@ -613,8 +613,7 @@ inline __device__ void PackLiteralsRoundRobin(
   auto scratch_bytes = reinterpret_cast<char*>(&scratch[0]);
   if (t < available_bytes) { dst[t] = scratch_bytes[t]; }
   if (t + 128 < available_bytes) { dst[t + 128] = scratch_bytes[t + 128]; }
-  // would need the following for up to 24 bits
-  // if (t + 256 < available_bytes) { dst[t + 256] = scratch_bytes[t + 256]; }
+  if (t + 256 < available_bytes) { dst[t + 256] = scratch_bytes[t + 256]; }
```

Comment:
Are these NUM_THREADS and NUM_THREADS*2?

Reply:
Yep.  More magic numbers to squash.

Reply:
Also fixed.